util/memory/tracker.go

```diff
@@ -238,7 +238,10 @@ func (t *Tracker) toString(indent string, buffer *bytes.Buffer) {
 	t.mu.Lock()
 	for i := range t.mu.children {
 		if t.mu.children[i] != nil {
+			tmp := t.mu.children[i].bytesLimit
+			t.mu.children[i].bytesLimit = 0
 			t.mu.children[i].toString(indent+"  ", buffer)
+			t.mu.children[i].bytesLimit = tmp
```

Comment:
Hi, 
I think it's not necessary to change this method, we can just print the Consumed and Limit memory of the Tracker itself.

Reply:
Actually I think so too, the code I changed is too ugly

Reply:
TestSetDDLErrorCountLimit success on my own machine...